libs/blocks/chart/chart.js

```diff
@@ -143,23 +175,35 @@ const init = async (el) => {
   children[1]?.classList.add('chart_subTitle');
   children[3]?.classList.add('chart_footnote');
 
-  // ToDo: Replace hardcoded size MWPW-112994
-  const size = LARGE;
+  // Add chart container
+  const container = document.createElement('section');
+  container.className = 'chart_container';
+  container.append(...children);
+  el.appendChild(container);
+
+  // Grab section metadata
+  const chartStyles = el.parentElement.classList;
+
```

Comment:
Unnecessary blank line
```suggestion
```

Reply:
I don't see that you removed this blank line. Variables shouldn't have extra lines in between them

Reply:
Got it, will remove.